src/main/java/pages/SignUpPage.java

```diff
@@ -0,0 +1,42 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SignUpPage {
```

Comment:
SignUpPage should be extends from Method handles class

Reply:
Done